src/v2/guide/components-registration.md

```diff
@@ -68,21 +68,21 @@ new Vue({ el: '#app' })
 </div>
 ```
 
-This even applies to all subcomponents, meaning all three of these components will also be available _inside each other_.
+Esto se aplica incluso a todos los subcomponentes, lo que significa que los tres componentes también estarán disponibles _dentro de cada uno_.
 
-## Local Registration
+## Registro local
 
-Global registration often isn't ideal. For example, if you're using a build system like Webpack, globally registering all components means that even if you stop using a component, it could still be included in your final build. This unnecessarily increases the amount of JavaScript your users have to download.
+El registro global a menudo no es ideal. Por ejemplo, si estás usando un sistema de compilación como Webpack, el registro global de todos los componentes significa que, incluso si dejas de usar un componente, podría ser incluido en tu compilación final. Esto aumenta innecesariamente la cantidad de JavaScript que sus usuarios tienen que descargar
```

Comment:
Webpack does not compile, it's a module bundler -- build system --. Knowing this, let's use:
```suggestion
El registro global a menudo no es ideal. Por ejemplo, si estás usando un empaquetador de módulos como Webpack, el registro global de todos los componentes significa que, incluso si dejas de usar un componente, podría ser incluido en tu compilación final. Esto aumenta innecesariamente la cantidad de JavaScript que sus usuarios tienen que descargar
```